plugins/Wallet/js/components/receiveprompt.js

```diff
@@ -1,15 +1,18 @@
 import React, { PropTypes } from 'react'
 
-const ReceivePrompt = ({address, actions}) => (
-	<div className="modal">
-		<div className="receive-prompt">
-			You can receive Siacoins using the following address:
-			<div className="wallet-address">{ address }</div>
-			<button className="receiveprompt-dismissbtn" onClick={actions.hideReceivePrompt}>OK</button>
+const ReceivePrompt = ({address, actions}) => {
+	const handleDismissClick = () => actions.hideReceivePrompt()
```

Comment:
why was this changed to a standalone function? Are we standardizing on "`onClick` handlers should be defined in the Component body?"